### ISSUE TITLE
[Legitimate Site Blocked] lilmutantapeclub.com

### ISSUE THREAD

**Issue by 0xYaper** [MetaMask/eth-phishing-detect#12591]
https://lilmutantapeclub.com/

this site is legitimate and it is not doing anything wrong. we are being harassed by a 3rd party.  please remove this notice. we are a legit nft project. this is hurting our brand

**Comment by AlexHerman1** [comment]
this domain appears to be suspended.

**Comment by 0xYaper** [comment]
Yes of course, We are currently dealing with a DMCA by yuga labs on our artwork and logo. We are not trademark infringing on them in any way. we are a derivative nft project that can not be confused with theirs in any way. Here is the example of the logo and a lil mutant. they are going around harrasing me every way they can. they have also filed with my domain hosting provider to take us down. please help 

![LMAC_logo](https://github.com/MetaMask/eth-phishing-detect/assets/100878691/96d01e10-5660-49bc-9ab8-69b7747779ee)
![IMG_1028](https://github.com/MetaMask/eth-phishing-detect/assets/100878691/673eec84-836b-4970-949e-f2815f726517)

**Comment by funkymonkey5150** [comment]
This is crazy yoga labs no bueno this will sure reCh the top this is crazy we got your back 0xyaper

**Comment by dubstard** [comment]
The warning flags are removed.

Seems like a zero effort cash grab.

"Why not just slap a few pixels on top of an existing project and call it a day."
